flink-libraries/flink-table/src/main/scala/org/apache/flink/table/expressions/stringExpressions.scala

```diff
@@ -358,6 +358,30 @@ case class Rpad(text: Expression, len: Expression, pad: Expression)
   }
 }
 
+/**
+  * Returns a string resulting from replacing all substrings
```

Comment:
Returns a string with all substrings that match the regular expression being replaced.